lib/quicly.c

```diff
@@ -5201,18 +5260,20 @@ int quicly_decrypt_address_token(ptls_aead_context_t *aead, quicly_address_token
                                  size_t len, size_t prefix_len, const char **err_desc)
 {
     const uint8_t *const token = _token;
-    uint8_t ptbuf[QUICLY_MAX_PACKET_SIZE];
+    uint8_t ptbuf[QUICLY_MIN_CLIENT_INITIAL_SIZE];
```

Comment:
Is this is an expectation we have of address tokens, to fit within the min size?  I don't see a similar assert in quicly_encrypt_address_token, perhaps that should be addded as well?

Reply:
We had that assertion, but it was hard to notice. Added comment in 8a06e0e.